src/components/PasswordInput.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState, type ComponentPropsWithRef } from "react";
+import { Input, InputGroup, InputRightElement, Button } from "@chakra-ui/react";
+
+type PasswordInputProps = ComponentPropsWithRef<typeof Input> & {
+  size?: "sm" | "md";
```

Comment:
[InputProps ](https://chakra-ui.com/docs/components/input/props) already have size, we shouldn't override it.

```suggestion
type PasswordInputProps =InputProps & {
```

Make sure to import InputProps from "@chakra-ui/react"